sideboard/configspec.ini

```diff
@@ -1,13 +1,28 @@
 
 debug = boolean(default=False)
 
+# Directory where core Sideboard pages look for plugins.
```

Comment:
Over time I'm trying to fill in code comments in places where they're missing.  I've gotten a lot better about filling them in for all new code as I go, but there's still a backlog of uncommented stuff back from my younger foolish days when I found comments too ugly to bear ruining my code with :P